v2/cmd/asoctl/pkg/importresources/find_child_resources_test.go

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT license.
+ */
+
+package importresources
```

Comment:
minor: Could be `importresources_test`, I think?

Cleaner as it avoids the temptation to test internals (which is sometimes what you want to do but sometimes not desirable).

Reply:
I'll move it, but really prefer to keep things close unless there's a problem (like a circular reference) that can be resolved by using the test package naming.